src/main/java/qna/domain/Answer.java

```diff
@@ -43,17 +46,22 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
-    public Answer setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
+    public DeleteHistory delete(final User user) {
+        verifyOwner(user);
+
+        deleted = Boolean.TRUE;
```

Comment:
`this` 예약어를 사용해 주시고요, 
특별한 이유가 없다면 primitive field에 wrapper type을 할당하는 것은 피해주세요.

```suggestion
        this.deleted = true;